uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-work-patterns.html

```diff
@@ -1,8 +1,8 @@
-<div class="btn-group">
+<p>
```

Comment:
I think it should a generic `<div>`, it would make more sense semantically

Reply:
While semantically yes, we need a margin on the bottom. I will have a look at other Bootstrap.

Reply:
Didn't find anything that may fulfill the requirements, so have to use custom style.